runtime/observability/telemetry-provider-otel/jvm/src/aws/smithy/kotlin/runtime/telemetry/otel/OtelTracerProvider.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package aws.smithy.kotlin.runtime.telemetry.otel
+
+import aws.smithy.kotlin.runtime.telemetry.context.Context
+import aws.smithy.kotlin.runtime.telemetry.trace.*
+import aws.smithy.kotlin.runtime.util.AttributeKey
+import aws.smithy.kotlin.runtime.util.Attributes
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.api.trace.Span as OtelSpan
+import io.opentelemetry.api.trace.SpanKind as OtelSpanKind
+import io.opentelemetry.api.trace.StatusCode as OtelStatus
+import io.opentelemetry.api.trace.Tracer as OtelTracer
+
+internal class OtelTracerProvider(private val otel: OpenTelemetry) : TracerProvider {
+    override fun getOrCreateTracer(scope: String, attributes: Attributes): Tracer =
+        OtelTracer(otel.getTracer(scope), attributes)
+}
+
+private class OtelTracer(
```

Comment:
**Suggestion**: This class has the same name as an import alias. I suggest changing it to `OtelTracerAdapter` or `OtelTracerImpl` or something.